<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>汽车销售管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>700</width>
      <height>410</height>
     </rect>
    </property>
    <widget class="QWidget" name="managePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>641</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>品牌车管理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>299</width>
         <height>267</height>
        </rect>
       </property>
       <attribute name="label">
        <string>出售车辆</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>厂家：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>54</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>品牌：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>报价：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>金额：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sellFactoryComboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="sellBrandComboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sellNumSpinBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>万</string>
        </property>
       </widget>
       <widget class="QLabel" name="sellLastNumLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>剩余数量：0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>万</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellSumLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>170</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sellOkBtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sellCancelBtn">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>299</width>
         <height>267</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QListWidget" name="dailyList">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>51</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="manageAction"/>
    <addaction name="chartAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>修改密码</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="manageAction">
   <property name="text">
    <string>品牌车管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="chartAction">
   <property name="text">
    <string>销售统计</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
